app/controllers/volunteer_ops_controller.rb

```diff
@@ -72,6 +72,9 @@ def volunteer_op_params
       :description,
       :title,
       :organisation_id,
+      :address,
+      :postcode,
+      :different_address
```

Comment:
ah, sorry I see - you're using it as a flag to indicate that we want it to have a different address.  I wonder if that flag is necessary?  

I would have thought we should keep the address and postcode as nil (which indicates same address as organisation) and then if they get edited directly and are then not nil, then that indicates that they have a different address.

I guess we have a related question there about whether we want changing organisation addresses to filter down automatically to volops ...